homeassistant/components/roborock/config_flow.py

```diff
@@ -153,3 +166,104 @@ def _create_entry(
                 CONF_BASE_URL: client.base_url,
             },
         )
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> config_entries.OptionsFlow:
+        """Create the options flow."""
+        return RoborockOptionsFlowHandler(config_entry)
+
+
+class RoborockOptionsFlowHandler(config_entries.OptionsFlowWithConfigEntry):
```

Comment:
I plan to add a lot more code to this to add 2 per-map forms for ColorPalette and ImageConfig; does it make sense to split this into a separate file?

Reply:
I've never seen a Options Flow in another file. That doesn't mean it never is, but I think it makes sense to just keep it in this file for now.